packages/server/src/lib/gost-aws.js

```diff
@@ -85,9 +85,13 @@ function sendEmail(message) {
         },
     };
     const command = new SendEmailCommand(params);
-    transport.send(command)
-        .then((data) => console.log('Success sending SES email:', JSON.stringify(data)))
-        .catch((err) => console.error('Error sending SES email:', err, err.stack));
+    try {
+        await transport.send(command)
+            .then((data) => console.log('Success sending SES email:', JSON.stringify(data)));
```

Comment:
This code will work, but is there a reason to mix the `await` and `.then` paradigm for async?

Reply:
Fixed.